docs/codeql/ql-language-reference/predicates.rst

```diff
@@ -226,14 +226,16 @@ Here are a few examples of infinite predicates:
       Compilation errors:
       ERROR: "i" is not bound to a value.
       ERROR: "result" is not bound to a value.
+      ERROR: expression is not bound to a value.
```

Comment:
Clarify which `expression` the error refers to. (It doesn't match the exact error message in the UI, but it makes sense to add extra context for our readers 😊)

```suggestion
      ERROR: expression "i * 4" is not bound to a value.
```